src/api/app/views/webui/request/_changes_content.html.haml

```diff
@@ -0,0 +1,12 @@
+- if diff_to_superseded_id
+  You're reviewing changes from
+  = link_to("superseded request ##{diff_to_superseded_id}.", request_show_path(number: diff_to_superseded_id))
```

Comment:
@ncounter we should stick to the bs_request number here. `id !=number`. Usually its the same, but its still two different things, and could lead to problems if we exchange them.

Reply:
ah its just the naming, you are using the number. but we should stay consistent with the naming here

Reply:
Exactly, it's only a variable name to the loop to keep the queryString parameter alive. The parameter is populated in [set_diff_to_superseded_id](https://github.com/openSUSE/open-build-service/pull/15887/files/950fb4d3a2202168031e5dd0bfab29e4a3ed87cc#diff-4240c3b55deedb4b829fef429bfc8fd6832885a1ed59928cabccd613a1494e18R356). 

I'm going to rename it. Thinking about it: before it was `diff_to_superseded`, now it is `diff_to_superseded_id`, I'm going for `diff_to_superseded_number` but I also wonder if I could get rid of the `diff_to` prefix and finalize it to `superseded_number` only :thinking: The fact that we use it for the `diff` is a detail, we could use it for something else too. What do you think?

Reply:
As discussed and decided offline, the renaming change will happen in a separate PR due to the fact the changes involves much more than the scope of this PR.

https://github.com/openSUSE/open-build-service/pull/15906